tekton/resources/nightly-tests/bastion-p/test_tekton_component.yaml

```diff
@@ -7,6 +7,9 @@ spec:
   - name: k8s-shared
     description: workspace to get k8s config file
     mountPath: /root/.kube
+  - name: registry-shared
+    description: workspace for docker config
+    mountPath: /root/.docker
```

Comment:
So, if I understand this right, this will be honoured by the go based e2e tests, but not by the yaml based tests, is that right?